scripts/g.search.modules/g.search.modules.py

```diff
@@ -311,10 +311,7 @@ def _exact_search(keyword, module_keywords):
     :param module_keywords: comma separated list of keywords
     """
     module_keywords = module_keywords.split(",")
-    for current in module_keywords:
-        if keyword == current:
-            return True
-    return False
+    return any(keyword == current for current in module_keywords)
```

Comment:
```suggestion
    return keyword in module_keywords
```

Reply:
Do we know if "keyword" is hashable? Are we sure it is a string?

Reply:
_module_keywords_ is a list of strings produced with `.split()`...

See also:  https://github.com/OSGeo/grass/blob/658a464ea259a2a9bb7d7ab30c587f4141314e22/scripts/g.search.modules/g.search.modules.py#L194